qdrant-landing/content/articles/modern-sparse-neural-retrieval.md

```diff
@@ -456,6 +457,25 @@ qdrant_client.upsert(
     ],
 )
 ```
+
+```python
+## LOCAL INFERENCE
```

Comment:
Not sure, whether to replace the old one or keep this next to it.

Reply:
When we reviewed the other articles with @generall , we decided that it is good enough to hide it under spoilers like here https://qdrant.tech/documentation/fastembed/fastembed-colbert/